vividus-extension-selenium/src/main/java/org/vividus/bdd/steps/ui/validation/BaseValidations.java

```diff
@@ -100,6 +100,14 @@ public boolean assertElementNumber(String businessDescription, String systemDesc
                 elementNumber(Matchers.comparesEqualTo(number)));
     }
 
+    @Override
+    public boolean assertElementsNumber(String description, List<WebElement> elements, ComparisonRule comparisonRule,
+            int number)
+    {
+        return uiContext.withAssertingWebElements(elements,
+            () -> softAssert.assertThat(description, elements.size(), comparisonRule.getComparisonRule(number)));
+    }
```

Comment:
elements, elementMatcher(comparison rule.getComparisonRule(size))

Reply:
Checked it and it turned out not to be properly aligned with comparison rule message, so we get as a result the following:
[Expected: number of elements is a value equal to <...> Actual: <...>]

Reply:
[Expected: number of elements is a value less than or equal to <...> Actual: <...>]